java-backend/src/main/java/org/kframework/backend/java/symbolic/SymbolicRewriter.java

```diff
@@ -1014,6 +1014,7 @@ private ConstrainedTerm applySpecRules(ConstrainedTerm constrainedTerm, List<Rul
             ConjunctiveFormula constraint = constrainedTerm.matchImplies(pattern, true, false,
                     new FormulaContext(FormulaContext.Kind.SpecRule, specRule), specRule.matchingSymbols());
             if (constraint != null) {
+                global.stateLog.log(StateLog.LogEvent.RULEATTEMPT, specRule.toKRewrite(), constrainedTerm.term(), constrainedTerm.constraint());
```

Comment:
Let's make a new event log `SRULEATTEMPT` for this, it may be useful to know that we're attempting a spec rule.